src/main/java/org/grizz/service/UserService.java

```diff
@@ -51,4 +71,20 @@ public User changePassword(String oldPassword, String newPassword) {
         }
         return currentUser;
     }
+
+    public User increaseNumberOfPlanetsAndSave(User currentUser) {
+        int actualPlanetNumber = currentUser.getPlanetNumber();
+        currentUser.setPlanetNumber(++actualPlanetNumber);
+        saveUser(currentUser);
+        return currentUser;
+    }
+
+    public User saveUser(User userToSave) {
+        log.debug("Saved user : ", userToSave);
```

Comment:
Why you share ability to save user outside userService. You are losing control of User entities in UserService